src/console-runners/seed-data.runner.ts

```diff
@@ -1,3 +1,4 @@
+import { generateSalt, hashPassword } from "#common/helpers/hash-password.helpers.js";
```

Comment:
use the barrel here, "import { generateSalt, hashPassword } from "#common/helpers/index.js";"